views/js/testRunner/actionBarHook.js

```diff
@@ -62,13 +63,18 @@ define([
     function initQtiTool($toolsContainer, id, toolconfig, assessmentTestContext){
 
         if(isValid(toolconfig)){
-            
+
+            var order = _.parseInt(toolconfig.order);
+            if(isNaN(order)){
```

Comment:
Use `_.isNan` in some platforms the behavior can be different (see https://github.com/lodash/lodash/blob/2.4.1/doc/README.md#_isnanvalue)